pkg/k8s/volumes/crud.go

```diff
@@ -192,6 +192,21 @@ func Destroy(ctx context.Context, name, namespace string, c *kubernetes.Clientse
 
 }
 
+// DestroyWithoutTimeout destroys a PVC without checking if it was detached or not
+func DestroyWithoutTimeout(ctx context.Context, name, namespace string, c kubernetes.Interface) error {
```

Comment:
In this case we don't have to wait until the volume is detached, so why it was called `WithoutTimeout`